src/services/autocomplete/autocompleteService_js.ml

```diff
@@ -64,7 +64,25 @@ let parameter_name is_opt name =
   let opt = if is_opt then "?" else "" in
   (Option.value name ~default:"_") ^ opt
 
+let lsp_completion_of_type (ty: Ty.t) =
+  let open Lsp.Completion in
+  match ty with
+  | Ty.InterfaceDecl _
+  | Ty.Inter _ -> Some Interface
+  | Ty.ClassDecl _ -> Some Class
+  | Ty.StrLit _
+  | Ty.NumLit _
+  | Ty.BoolLit _ -> Some Value
+  | Ty.Fun _ -> Some Function
+  | Ty.TypeAlias _
+  | Ty.Tup _ -> Some Variable
+  | Ty.Union _ -> Some Enum
+  | Ty.Module _ -> Some Module
+  | _ -> Some Variable
```

Comment:
Hi @vicapow! Can you expand the rest of the cases here (instead of the catch-all), so that we don't miss something when the Ty.t is updated?